app/lib/default_bookable_slots.rb

```diff
@@ -0,0 +1,38 @@
+class DefaultBookableSlots
+  Instance = Struct.new(:date, :start, :end)
+
+  attr_reader :from
+  attr_reader :to
+
+  def initialize(from = Date.current, to = from.advance(weeks: 6))
+    @from = from
+    @to = to
+  end
+
+  def call
+    available_days.map do |date|
+      [
+        Instance.new(date.iso8601, '0900', '1300'),
+        Instance.new(date.iso8601, '1300', '1700')
+      ]
+    end.flatten
+  end
+
+  private
+
+  def available_days
+    (grace_period..to).reject { |day| day.on_weekend? || bank_holiday?(day) }
+  end
+
+  def bank_holiday?(date)
+    BANK_HOLIDAYS.include?(date)
+  end
+
+  def grace_period
```

Comment:
Would something like `end_of_grace_period` describe the date this produces more accurately?